Kelompok 7 RA Tugas Kelompok Artikel Modul 2, 3, 4, 5, 6, 7

alkhrzmy opened the pull request:
Kelompok 7 RA
- Tessa Sagala Kania 122450040
- Renisha Putri Giani 12245079
- Rian Bintang Wijaya 122450094
- Gymnastiar Al Khoarizmy 122450096
- Virdio Samuel Saragih 122450124

alkhrzmy committed:
Add files via upload

alkhrzmy committed:
Rename Kelompok07_ArtikelModul02PBF.docx.pdf to Kelompok7_RC_Modul2_PrakPBF.pdf

alkhrzmy committed:
Rename Kelompok7_RC_Modul2_PrakPBF.ipynb to Kelompok7_RA_Modul2_PrakPBF.ipynb

alkhrzmy committed:
Rename Kelompok7_RC_Modul2_PrakPBF.pdf to Kelompok7_RA_Modul2_PrakPBF.pdf

alkhrzmy committed:
Add artikel modul 3 and ipynb for the code

diosamuel commented:
Pull request ini berisi 
## Modul 2, 3, 4, dan 5 Kelompok 7 RA.
Untuk modul 1 ada di pull request saya (Virdio Samuel) pada link https://github.com/sains-data/Praktikum-SD2201-Pemrograman-Berbasis-Fungsi/pull/17

Terimakasih.

alkhrzmy committed:
Artikel Modul 4 dan Ipynb

alkhrzmy committed:
Artikel Praktikum PBF Modul 5 dan IPYNB Kelompok 7 RA

Kelompok 7 RA
Tessa Sagala Kania 122450040
Renisha Putri Giani 12245079
Rian Bintang Wijaya 122450094
Gymnastiar Al Khoarizmy 122450096
Virdio Samuel Saragih 122450124

alkhrzmy committed:
Artikel Modul 6

Kelompok 7 RA

alkhrzmy committed:
Add files via upload
